dali/c_api/c_api_test.cc

```diff
@@ -659,25 +697,24 @@ TYPED_TEST(CApiTest, ForceNoCopyFail) {
   auto pipe_ptr = GetExternalSourcePipeline(false, other_device_str);
   auto serialized = pipe_ptr->SerializeToProtobuf();
 
-  pipe_ptr->Build();
-
   daliPipelineHandle handle;
   daliCreatePipeline(&handle, serialized.c_str(), serialized.size(), batch_size, num_thread,
                      device_id, false, prefetch_queue_depth, prefetch_queue_depth,
                      prefetch_queue_depth, false);
 
-  SequentialFill(view<uint8_t>(input_cpu), 42);
-  // Unnecessary copy in case of CPUBackend, makes the code generic across Backends
-  input.Copy(input_cpu, cuda_stream);
+    SequentialFill(TensorListView<StorageCPU, uint8_t>(input_cpu.get(), input_shape), 42);
+    // Unnecessary copy in case of CPUBackend, makes the code generic across Backends
+    MemCopy(input.get(), input_cpu.get(), num_elems, cuda_stream);
```

Comment:
Hmmm... this seems to make _more_ assumptions about buffer contiguity, not less.